cdap-security/src/main/java/co/cask/cdap/security/authorization/AbstractAuthorizationService.java

```diff
@@ -105,8 +105,8 @@ protected void startUp() throws Exception {
   @Override
   protected void runOneIteration() {
     if (!isAuthCacheEnabled()) {
-      LOG.debug("Authorization cache is disabled security: {}; authorization: {}; cache: {}",
-               securityEnabled, authorizationEnabled, cacheEnabled);
+      LOG.trace("Authorization cache is disabled security: {}; authorization: {}; cache: {}",
```

Comment:
Its missing a period after "disabled". Authorization cache could be disabled because either security, authorization or caching is disabled.